vllm/utils.py

```diff
@@ -77,11 +77,9 @@ def touch(self, key: Hashable) -> None:
         self.cache.move_to_end(key)
 
     def get(self, key: Hashable, default_value: Optional[Any] = None) -> int:
-        if key in self.cache:
-            value = self.cache[key]
+        value = self.cache.get(key, default_value)
+        if value is not default_value:
```

Comment:
The value might be the same as default_value, and the key can not be update in this case.

```python
# value is boolean and key in cache
value = cache.get(key, False)
```

Reply:
thanks for your reminding